web/store/pages/project-page.store.ts

```diff
@@ -81,28 +70,30 @@ export class ProjectPageStore implements IProjectPageStore {
 
   /**
    * @description get the current project page ids based on the pageType
+   * @param {TPageNavigationTabs} pageType
    */
-  get currentProjectPageIds() {
+  currentProjectPageIds = computedFn((pageType: TPageNavigationTabs) => {
```

Comment:
Just one minor change, can you rename it to getCurrentProjectPageIds since it is a method now